games-action/powermanga/powermanga-0.93.1.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="A 2D arcade shoot-em-up game"
+DESCRIPTION="Powermanga is an arcade 2D shoot-em-up game with 41 levels and more than 200 sprites.
```

Comment:
Lines mustn't be linger than 80 chars. add a "\" at the end to escape a newline.